src/components/CuratorDashboard/CuratorProfile/CuratorProfileCard.tsx

```
@@ -0,0 +1,218 @@
+		if (!loginAddress) return;
+		const substrateAddress = getSubstrateAddress(loginAddress);
+		const { data, error } = await nextApiClientFetch<IGetProfileWithAddressResponse>(`api/v1/auth/data/profileWithAddress?address=${substrateAddress}`);
+		if (data) setCuratorProfile(data);
+		if (error) {
+			console.log(error, 'error');
+			setCuratorProfile(null);
+		}
+	};
+
+	const fetchCuratorBio = async () => {
+		if (!id || isNaN(id) || id < 0) return;
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)